bin/default.nix

```diff
@@ -0,0 +1,54 @@
+{ pkgs, writeShellApplication }:
+{
+  generate-coverage = writeShellApplication {
+    name = "generate-coverage";
+
+    runtimeInputs = with pkgs; [
+      cacert
+      grcov
+      rustup
+    ];
+
+    text = builtins.readFile ./generate-coverage.sh;
+  };
```

Comment:
Does this still work? I remember getting a working environment to generate the coverage data was quite tricky and it didn't play well with the normal Topiary development environment, as it needs `rustup` to install `llvm-tools-preview`.

Reply:
No, it doesn't you are right. I think I'll just remove this script in its entirety. Or refactor it perhaps when I have the time. I'll move this PR into a Draft state for now.

Thank you <3

Reply:
AFAIK, no one is using it. I think this PR is valuable, so this being broken probably shouldn't block things... Maybe just comment it out with a `FIXME`?